src/library/validators.js

```diff
@@ -13,12 +18,12 @@ const containNumber = str => /[0-9]/.test(str)
 const containSpecialCharacter = str => /[!?$#@()\-*]/.test(str)
 
 const getPasswordInfo = str => ({
+  minLength: str.length >= 6 ? 6 : 0,
```

Comment:
I think we can simplify this by just returning 1 or 0.
This way, the max score corresponds to the number of properties of the object returned by `getPasswordInfo`

Reply:
It can also be simpler if you don't use a ternary test:
`secureLength: str.length >= 12` for instance, you just need to modify computeScore to check for `true (1)` values

Reply:
+1, done.